### DISCUSSION TITLE
Fraud List Blocking Sites?

### DISCUSSION THREAD

**Discussion by ericewers** [blocklistproject/Lists#1035] [General]
I went to check why a ton of different random websites I try to load are not working, and it looks like the Fraud list (https://blocklistproject.github.io/Lists/fraud.txt) is the culprit.

YouTube, Netflix, Hulu, MSN, Disney Plus, Discord, SpeedTest, Fast.com, etc.

Can anyone else confirm this? I unchecked the Fraud list in my custom filters, and everything started loading again.

**Comment by gap579137** [comment]
There was a scripting issue. Please update your lists and you should be back online.
